cycledash/static/js/examine/components/VCFTable.js

```diff
@@ -332,10 +331,11 @@ var VCFRecord = React.createClass({
     ];
     _.each(this.props.columns, (columns, topLevelColumnName) => {
       for (var columnName in columns) {
-        var column = columns[columnName];
+        var column = columns[columnName],
+            val = this.props.record[column.path.join(':')];
         tds.push(
-          <td key={column.path.join(':')} title={column.path.join('→')}>
-            {this.formatCell(column)}
+          <td key={column.path.join(':')} title={val}>
```

Comment:
Now that we're truncating values, if they're too long, it seemed important to show those instead of showing the name of the column.